### PR TITLE
WIP:  Stakes table UI

### PR DESCRIPTION
**Pull request by georgeweiler**
Screen Shot 2022-01-13 at 10 31 50 PM

### CODE REVIEW

File: src/pages/Staking/StakesTable/index.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+              <Tr key={stake.operator}>
+                <StakeNameCell stake={stake} index={i} />
+                <StakeAddressesCell stake={stake} />
+                <StakeBalanceCell stake={stake} />
+                <StakeActionsCell stake={stake} />
+              </Tr>
+            )
+          })}
+        </Tbody>
+      </Table>
```
Comment (r-czajkowski) [inline src/pages/Staking/StakesTable/index.tsx:38]: We should probably create a component wrapper for DataTable. something like:

```
<DataTable data={<data here>} >
 <Column header={"name"} fieldId={"operator"}>
  <render content here>
</Column>
<Column header={"addresses"} fieldId={"operator"}>
  <render content here>
</Column>
...
</DataTable>
```

And the `DataTable` should be responsible for mapping data into table cells etc. Ofc let's address it in a followup work but we can just leave a TODO.

File: src/pages/Staking/index.tsx [hunk 2]
```diff
@@ -26,6 +27,8 @@ const StakingPage: FC = () => {
     fn()
   }, [fetchOwnerStakes, account, dispatch])
 
+  console.log("stakes", stakes)
```
Comment (r-czajkowski) [inline src/pages/Staking/index.tsx:30]: Let's remove it.

File: src/utils/getUsdBalance.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+const getUsdBalance = (
+  balance: string | number,
+  usdConversion: number
+): string => {
+  return numeral(
+    FixedNumber.fromString(usdConversion.toString())
+      .mulUnsafe(FixedNumber.fromString(formatUnits(balance)))
+      .toString()
+  ).format("$0,0.00")
```
Comment (r-czajkowski) [inline src/utils/getUsdBalance.tsx:13]: Let's use `formatFiatCurrencyAmount` util function.

**Review by r-czajkowski — APPROVED** [review]
LGTM! just left minor comments

File: src/pages/Staking/StakesTable/index.tsx (hunk 1 above)
Comment (georgeweiler) [reply src/pages/Staking/StakesTable/index.tsx:38]: I made a quick ticket here to capture this: https://keepnetwork.atlassian.net/browse/DAPP-516

**Pull request merged**